server/api/assessment/index.ts

```diff
@@ -33,6 +34,13 @@ export const AssessmentApi = {
     express.get(ApiEndPoint.Assessment.OriginalDataPoint.one(), AuthMiddleware.requireView, getOriginalDataPoint)
     express.put(ApiEndPoint.Assessment.OriginalDataPoint.one(), AuthMiddleware.requireEdit, updateOriginalDataPoint)
 
+    // OriginalDataPoint // table
+    express.get(
+      ApiEndPoint.Assessment.OriginalDataPoint.TableData.one(),
+      // AuthMiddleware.requireView,
```

Comment:
this should be added. in case we dont have section, add AuthMiddleware.requireOriginalDataPointView that calls AuthMiddleware.requireView with section extentOfForest

Reply:
Was removed for easier testing, will implement